target/linux/ath79/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -152,31 +152,33 @@ case "$FIRMWARE" in
 	;;
 "ath9k-eeprom-pci-0000:00:11.0.bin")
 	case $board in
-	dlink,dir-825-b1)
-		ath9k_eeprom_extract "caldata" 4096 3768
-		ath9k_patch_fw_mac_crc $(mtd_get_mac_text "caldata" 65440) 524
-		;;
+	buffalo,wzr-hp-ag300h|\
 	netgear,wndr3700|\
 	netgear,wndr3700v2|\
 	netgear,wndr3800)
 		ath9k_eeprom_extract "art" 4096 3768
 		;;
+	dlink,dir-825-b1)
+		ath9k_eeprom_extract "caldata" 4096 3768
+		ath9k_patch_fw_mac_crc $(mtd_get_mac_text "caldata" 65440) 524
+		;;
 	*)
 		ath9k_eeprom_die "board $board is not supported yet"
 		;;
 	esac
 	;;
 "ath9k-eeprom-pci-0000:00:12.0.bin")
 	case $board in
-	dlink,dir-825-b1)
-		ath9k_eeprom_extract "caldata" 20480 3768
-		ath9k_patch_fw_mac_crc $(macaddr_add $(mtd_get_mac_text "caldata" 65460) 1) 524
-		;;
+	buffalo,wzr-hp-ag300h|\
 	netgear,wndr3700|\
 	netgear,wndr3700v2|\
 	netgear,wndr3800)
 		ath9k_eeprom_extract "art" 20480 3768
 		;;
+	dlink,dir-825-b1)
```

Comment:
Probably an oversight - the D-Link board should be listed after your new Buffalo entry, not after the Netgear ones (alphabetically).

Reply:
The Buffalo device uses the same code as the netgear devices, that's why the dlink was effectively moved after the netgear devices, because buffalo sorts before dlink. The diff looks like the dlink was added after the netgear devices, but in fact it stayed where it is, the buffalo device was added above it, and the netgear devices were moved into the same case group as the new buffalo.

I gathered from past commits and existing sources that basically only the first of multiple devices in the same case group is used for ordering, and groups should not be split up just be up just because another device would be sorted inbetween.

Reply:
Yes, the way it is done is correct.

Reply:
Yep nevermind me, totally overlooked that those were different entries :-(.